weblate_web/templates/snippets/service.html

```diff
@@ -1,94 +1,161 @@
-                        {% if not service.check_in_limits %}
-                        <div class="alert error">
-                        {% trans "Exceeds selected service offering." %}
-                        </div>
-                        {% endif %}
+                    {% for owner in service.users.all %}
+                    <form id="server_user_{{owner.id}}_form" method="post" action="{% url 'service-user' pk=service.pk %}">
+                    <div class="user-slot">
+                        <div class="user-slot-content">
+                        <p>{{ owner.last_name }}</p>
+                    <t>{{ owner.email }}</t></div>
+                    {% if owner == user %}
+                    <div class="user-img"><img src="{% static "img/Shield.png" %}" alt="Shield"></div>
+                    {% else %}
+                    {% csrf_token %}
+                    <img class="user-img" onclick="removeUser('{{owner.id}}')" name="remove" value="remove" src="{% static "img/delete.png" %}">
+                    {% endif %}
+                </div>
+                    {% csrf_token %}
+                    {% if owner != user %}
+                    <input type="hidden" name="email" value="{{ owner.email }}" />
+                    <input type="hidden" class="" name="remove" value="{% trans "Remove" %}" />
+                    {% endif %}
+                    </form>
+                    {% endfor %}
+                    <form method="post" action="{% url 'service-user' pk=service.pk %}">
+                    {% csrf_token %}
+                    <input type="email" placeholder="{% trans "Invite e-mail" %}" name="email" />
+                    <input type="submit" class="button med1 inline border rev" value="{% trans "Add user" %}" />
+                    </form>
                     </div>
                     <div class="clear"></div>
                 </div>
+
                 {% if service.needs_token %}
-                <div class="form-line">
-                    <div class="line-left">{% trans "Activation token" %}</div>
+                <div class="form-line style6 bdr-btm">
+                    <div class="line-left pt-1">{% trans "Activation token" %}</div>
                     <div class="line-right">
                         <input value="{{ service.secret }}" readonly class="fullwidth"><br />
-                        <a class="button small inline" data-clipboard-text="{{ service.secret }}">{% trans "Copy" %}</a>
+                        <a class="button med3 inline" data-clipboard-text="{{ service.secret }}">{% trans "Copy" %}</a>
                         <form class="form-inline" method="post" action="{% url "service-token" pk=service.pk %}">
                         {% csrf_token %}
-                        <input type="submit" class="button small inline border rev" value="{% trans "Regenerate" %}">
+                        <input type="submit" class="button med4 inline border rev" value="{% trans "Refresh code" %}">
```

Comment:
The row is called token, so be consistent here:

```suggestion
                        <input type="submit" class="button med4 inline border rev" value="{% trans "Refresh token" %}">
```

Reply:
It's done